### PR TITLE
stalebot: add exempt label

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**

**What this PR does / why we need it**:

Now some issues and prs may need more time to discuss or review, add an exempt label to reduce the wrong close.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ onlyLabels: []
 exemptLabels:
   # This label is hardcoded on remind bot (https://probot.github.io/apps/reminders/) and is used by remind bot when
   # issue is being reminded.
-  - reminder
+  # - reminder
+  # This label is added by commenting with "/lifecycle frozen" by anyone
+  - lifecycle/frozen
```
Comment (fisherxu) [inline .github/stale.yml:14]: User need to add `/lifecycle frozen` or `lifecycle/frozen`?

Comment (daixiang0) [reply .github/stale.yml:14]: Now some issues and prs may need more time to discuss or review, add an exempt label to reduce tips.

Comment (fisherxu) [reply .github/stale.yml:14]: The format of label, I think should be `lifecycle/frozen` as the defined.

Comment (daixiang0) [reply .github/stale.yml:14]: It exists as https://github.com/kubeedge/kubeedge/labels/lifecycle%2Ffrozen.

**Comment by kevin-wangzefeng** [comment]
Seems the DCO check is not responding

**Comment by daixiang0** [comment]
@kevin-wangzefeng done.

**Pull request merged**